cypress/support/pages/course/pageCourses.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+class Courses {
+  static #createFAB = '[name="fab-icon"]'
+  static #courseTitle = '[data-testid="coursename"]'
+  static #nextButton = '[id="nextSection"]'
+  static #nextContinueButton = '[data-submit-label="Kurs anlegen und Weiter"]'
+  static #goToCourseOverviewButton = '[data-testid="zur-uebersicht-btn"]'
+  //static #threeDotButton = '[type="button"]'
+  //static #editOption = '[class="three-dot-button v-btn v-btn--icon v-btn--round theme--light v-size--default"]'
+  //static #selectOption = '[class="v-icon notranslate menu-action-icon mr-1 theme--light"]'
+  static #deleteButton = '[data-method="DELETE"]'
+  static #confirmDeletionPopup = '[data-testid="modal_delete_course_button"]'
+  static #pageTitle = '[id="page-title"]'
+  static #dropDownCourse = '.course-title .three-dot-button'  //'.three-dot-button:first-of-type'//'[data-testid="title-menu"]'
+  static #btnCourseEdit = '[data-testid="title-menu-edit-delete"]'
+  static #selectClassesInput = '#classId_chosen > ul > li > input'
+  //static #selectClassesDropdown = '#classId_chosen > div > ul > li:nth-child(1)'
+  static #selectClassesDeleteButtons = '#classId_chosen > ul > li > a'
+  static #selectClasses = '#classId_chosen > ul > li > span'
+  static #btnSubmit = '#main-content > section > form > div.modal-footer > button.btn.btn-primary.btn-submit'
+  static #classSearchString = 'ALLE'
+  static #numberStudentsInClass = 2
+  static #selectStudents = '#studentsId_chosen > ul > li'
+
+  clickOnCreateFAB() {
+    cy.get(Courses.#createFAB).click()
+  }
+
+  fillCourseCreationForm(new_course) {
+    cy.get(Courses.#courseTitle).type(new_course)
+  }
+
+  clickOnNextSteps() {
+    cy.get(Courses.#nextButton).click()
+    cy.get(Courses.#nextContinueButton).click()
+    cy.get(Courses.#goToCourseOverviewButton).click()
+  }
+
+  performDeletion() {
+    cy.get(Courses.#deleteButton).click()
+    cy.get(Courses.#confirmDeletionPopup).click({
+      multiple: true,
+      force: true
+    })
+  }
+
+  // edit course
+  openCourseEditPage() {
+    cy.get(Courses.#dropDownCourse).click()
+    cy.get(Courses.#btnCourseEdit).click()
+  }
+
+  showCourseEditPage() {
+    cy.get(Courses.#pageTitle)
+    cy.contains('Kurs bearbeiten')
+  }
+
+  selectClassForCourse() {
+    //type.('{enter}') simulates hitting 'enter' button from the keyboard to select class ALLE from the dropdown
```

Comment:
Remove it as well and below all the commented lines

Reply:
removed comment in line 60 here, below comments in this file are in main already so we can remove them in a followup up, because that is logic explanation for the drop down behavior in curse edit page.